### ISSUE TITLE
v-select doesn't have a placeholder

### ISSUE THREAD

**Issue by PELEk** [vuetifyjs/vuetify#1382]
### Steps to reproduce
Set up placeholder prop on `v-select`

### Versions
0.14.8

### What is expected?
Placeholder should be visible like in `v-text-field`.

### What is actually happening?
Placeholder is missing.

### Reproduction Link
https://codepen.io/bartoszmagier/pen/MvzgGG

**Comment by flybayer** [comment]
In the documentation, placeholder is a prop for v-select, but it's not actually working for me.

**Comment by johnleider** [comment]
When creating an issue you should be provided with boilerplate for the ticket. This is important for myself and others to properly address your issue. Please reopen this with the required information, thank you.

**Comment by Lirt** [comment]
There is actually a typo and placeholder is `placholder` (Props - https://vuetifyjs.com/components/selects). But it is not working for me either.

**Comment by PELEk** [comment]
@johnleider Please reopen the issue.

**Comment by Lirt** [comment]
Currently I am able to setup default values (placeholders ) with this sample code [https://codepen.io/anon/pen/vJQYOy](https://codepen.io/anon/pen/vJQYOy). So setting `v-model` to some `data` variable and changing it for example in `mounted` works fine. Note that **item you are setting as default must exist** in items `bind`. 

When I tried to use `placeholder`/`placholder` instead of model as in @PELEk sample, it does not work - https://codepen.io/anon/pen/GvwgYa

The way this can be most usable in my opinion is if we can use `computed` value or `vuex getter` in `placeholder` argument. Or second option if we were able to use `computed` value or `vuex getter` in `v-model` (not sure if this is not against vue architecture). 

The reason is we can get placeholder value from complex nested objects located in vuex store or wherever.

Example use

template:
```js
<v-select
  v-bind:items="selector-items"
  label="Item selector"
  placeholder="getSelectorPlaceholder"
></v-select>
```
script:
```js
data () {
  return {
    selector-items: [{
      'text': 'item1',
      'value': '1'
      }, {
      'text': 'item2',
      'value': '1'
      }
    ],
    placeholderItem: '2' //find value in selector items
  }
}
computed: {
  getSelectorPlaceholder: function () {
    return this.placeholderItem
  }
}
```
